di-ipv-credential-issuer-stub/README.MD

```diff
@@ -14,22 +15,39 @@ The stub application to simulate the credential issuers to be used during the IP
 | CREDENTIAL_ISSUER_PORT  | The port number the credential issuer should run on   | `8084`                 |
 | CREDENTIAL_ISSUER_NAME  | The name of the credential issuer, displayed to users | `UK Passport CRI Stub` |
 | CREDENTIAL_ISSUER_TYPE  | The name of the credential issuer                     | `CRI`                  |
-| CLIENT_CONFIG           | Configuration of accepted clients                     | `{"foo":"bar"}`        |
-
+| CLIENT_CONFIG           | Configuration of accepted clients                     | See [test fixtures](https://github.com/alphagov/di-ipv-stubs/blob/main/di-ipv-credential-issuer-stub/src/test/java/uk/gov/di/ipv/stub/cred/fixtures/TestFixtures.java#L6) for example values                       |
+| CLIENT_AUDIENCE         | Hostname that the credential issue should run on      | `http://localhost:8084` |
+| VC_ISSUER               | Hostname of the VC issuer                             | `http://localhost:8084` |
+| VC_SIGNING_KEY          | Signing key for the VC issuer | See [test fixtures](https://github.com/alphagov/di-ipv-stubs/blob/main/di-ipv-credential-issuer-stub/src/test/java/uk/gov/di/ipv/stub/cred/fixtures/TestFixtures.java#L6) for example values      |
 
 ## Running locally
 
+### Startup script
+
 To run locally, you can run the startup script.
+
 ```shell
 ./startup.sh
 ```
+
 This will first build the project with gradle,
-and after it will attempt to run it.
+and after it will attempt to run it. Java 17 is required.
 
 Once started, visit the `/authorize` endpoint with the required query params
  [http://localhost:8084/authorize?response_type=code&client_id=test&redirect_uri=http://example.com](http://localhost:8084/authorize?response_type=code&client_id=test&redirect_uri=http://example.com).
 
 By clicking on the `Generate Auth Code` button you should be redirected to the given redirect_uri with an authentication code as a query param.
```

Comment:
This probably needs updating - could mention the JWT `request` param and can remove line 39